subworkflows/nf-core/vcf_validate_small_variants/main.nf

```diff
@@ -0,0 +1,185 @@
+include { RTGTOOLS_VCFEVAL      } from '../../../modules/nf-core/rtgtools/vcfeval/main'
+include { RTGTOOLS_ROCPLOT      } from '../../../modules/nf-core/rtgtools/rocplot/main'
+include { HAPPY_HAPPY           } from '../../../modules/nf-core/happy/happy/main'
+
+workflow VCF_VALIDATE_SMALL_VARIANTS {
+
+    take:
+    ch_vcf                          // [mandatory] channel: [ meta, vcf, tbi, truth_vcf, truth_tbi, regions_bed, targets_bed ]
```

Comment:
I would split the truth vcf into a separate channel with it's own meta.
The bed files can stay with the sample vcf IMO

Reply:
Although, I think it's better to keep them together as all modules run like this and this way the user can specify themselves which truth sets belong to what query set without having to fiddle with the meta too much

Reply:
fine by me 🤷🏻

Reply:
Good point! I'll change that :)

Reply:
Fixed in the last commit